irods_specific_implementation/utils.hpp

```diff
@@ -63,9 +62,10 @@ namespace irods::rest
         _response.send(http_code, msg);
     } // handle_request
 
-    inline bool to_bool(const std::string_view s){
+    inline bool to_bool(const std::string_view s)
+    {
         return s == "1"; // we only honor "1" for this client
-    } // to_bool
+    }                    // to_bool
```

Comment:
Am I the only one that thinks this is goofy? :p 

Also, is `to_bool` being used anywhere?

Reply:
I was kind of wondering about this. If it looks silly in the context in which we removed it, should we just nix the function entirely?

Reply:
Ah, I was referring to clang-format aligning the comments. The function itself is fine if it's useful. However, if `to_bool` is not being used then I say we torch it.

Reply:
Yes, let's remove this function.